src/components/dashboard/components/dictionary/ListDictionaries.jsx

```diff
@@ -15,40 +15,27 @@ const ListDictionaries = ({
   }
   if (dictionaries.length >= 1) {
     return (
-      <div>
-        { dictionaryViewOptions === 'user' ?
-          <div className="row justify-content-center">
-            {dictionaries.map((dictionary, index) => {
-            return dictionary.created_by === localStorage.getItem('username')
-              && dictionaryViewOptions === 'user' ?
-                <Card
-                  dictionary={dictionary}
-                  key={dictionary.uuid}
-                  fetchData={fetchData}
-                />
-              :
-                <div key={index}> {' '} </div>;
-          })}
-          </div>
-        :
-          <div className="row justify-content-center">
-            {dictionaries.map((dictionary, index) => {
-            return dictionary.repository_type === 'OpenMRSDictionary'
-              && dictionaryViewOptions === 'organizations' ?
-                <Card
-                  dictionary={dictionary}
-                  key={dictionary.uuid}
-                  fetchData={fetchData}
-                />
-              :
-                <div key={index}> {' '} </div>;
-          })}
-          </div>
-        }
+      <div className="row justify-content-center">
+        {dictionaries.map((dictionary, index) => (dictionary.repository_type === 'OpenMRSDictionary' ?
```

Comment:
This filtering could be done in the reducer, that way it will be easier to paginate the items in the future.

Reply:
Thank you @emasys for the feedback.
I have made the relevant corrections.